app/controllers/admin/events_controller.rb

```diff
@@ -94,7 +94,9 @@ def edit
     end
 
     def comment
-      comment = Comment.build_from(@event, current_user.id, comment_params)
+      comment = Comment.new(comment_params)
+      comment.commentable = @event
+      comment.user_id = current_user.id
```

Comment:
In fact, we could also eliminate user_id from controller and pass it with a hidden field from the view form.